documentation/asciidoc/computers/configuration/host-wireless-network.adoc

```diff
@@ -0,0 +1,165 @@
+2. Click the network icon in the system tray. Select *Turn On Wireless LAN*.
+
+The wireless module should automatically connect to your preferred wireless network when you complete the second step. If it doesn't, pick a wireless network from the dropdown.
+
+==== Command line
+
+To create a hosted wireless network on the command line, run the following command, replacing the `<example-network-name>` and `<example-password>` placeholders with your own values:
+
+----
+sudo nmcli device wifi hotspot ssid <example-network-name> password <example-password>
+----
+
+To disable the network, run the following command:
+
+----
+sudo nmcli device disconnect wlan0
+----
+
+After disabling the network, run the following command to reconnect to another Wi-Fi network:
+
+----
+sudo nmcli device up wlan0
+----
+
+TIP: For more information about connecting to wireless networks, see xref:configuration.adoc#configuring-networking[Configuring Networking].
+
+=== Use your Raspberry Pi as a network bridge
+
+By default, the wireless network hosted from your Raspberry Pi exists separately from the parent network connected via Ethernet. In this arrangement, devices connected to the parent network cannot directly communicate with devices connected to the wireless network hosted from your Raspberry Pi. If you want connected wireless devices to be able to communicate with devices on the parent network, you can configure your Raspberry Pi as a https://en.wikipedia.org/wiki/Network_bridge[network bridge]. With a network bridge in place, each device connected to the Pi-hosted wireless network is assigned an IP address in the parent network.
+
+////
+In the diagram below, note that the laptop exists in the same IP block as the router and wired clients:
+
+----
+                                         +- RPi -------+
+                                     +---+ 10.10.0.2   |          +- Laptop ----+
+                                     |   |     WLAN AP +-)))  (((-+ WLAN Client |
+                                     |   |  Bridge     |          | 10.10.0.5   |
+                                     |   +-------------+          +-------------+
+                 +- Router ----+     |
+                 | Firewall    |     |   +- PC#2 ------+
+(Internet)---WAN-+ DHCP server +-LAN-+---+ 10.10.0.3   |
+                 |   10.10.0.1 |     |   +-------------+
+                 +-------------+     |
+                                     |   +- PC#1 ------+
+                                     +---+ 10.10.0.4   |
+                                         +-------------+
+----
+TODO: Switch to an image version of this ascii art!
+////
+
+The following steps describe how to set up a network bridge on your Raspberry Pi to enable communication between wireless clients and the parent network.
+
+First, create a network bridge interface:
+
+----
+sudo nmcli connection add type bridge con-name 'Bridge' ifname bridge0
+----
```

Comment:
Can this _only_ be done via the command-line then, in contrast to the non-bridged network? If that's the case, it might be worth mentioning that, so that people don't think that the GUI instructions are "missing" ?